docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+### Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
+|:----------------|:-----------------------------------------|
+| Single Type     | `POST /{content-type}/:id/localizations` |  
+| Collection Type | `POST /{content-type}/localizations`     |
+
+
+::: warning CAUTION
+When creating a localization for existing localized entries, the body of the POST request can only accept localized fields.
+:::
+
+When sending the request to a collection type, Strapi will:
+
+1. use the `id` as a base entry for the non-localized fields and copy them in the new entry
+2. then create a new entry for the given locale and link it with the base entry.
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants/8/localizations
+```
+
+```json
+{
+  "locale": "en",
+  "name": "Restaurant F",
+  "description": "Restaurant facing the sea"
+}
+```
+
+This query:
+
+1. creates a new entry in `en`
+2. links the created entry with `restaurant:8` (they will share the same `localizations` object)
+3. copies every non-localized fields from `restaurant:8` into the new entry and keeps the localized fields from the request's body
+
+:::
+
+::: tab Response
+
+**Example reponse**
+
+```json
+{
+    "id": 9,
+    "name": "Restaurant F",
+    "slug": null,
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant facing the sea",
+    "menu": null,
+    "locale": "en",
+    "published_at": "2021-04-08T13:29:53.295Z",
+    "created_at": "2021-04-08T13:29:53.297Z",
+    "updated_at": "2021-04-08T13:29:53.297Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": [
+        {
+            "id": 8,
+            "locale": "fr",
+            "published_at": "2021-04-07T13:22:46.589Z"
+        }
+    ]
+}
+```
+
+:::
+
+::::
+
+<!-- ? should we also add an example of creating a localization for a collection type? -->
+
+::: tip TIP
+The Content-Type should have the [`createlocalization` permission]() enabled, otherwise the POST request will return a `403: Forbidden` status.
+<!-- TODO: insert cross-link to user guide on how to activate `createlocalization` permission in the admin panel -->
+:::
+
+### Updating an entry
+
+Currently, it is not possible to change the locale of an existing localized entry using the Content API.
```

Comment:
I don't think it's possible to do it either from the Content Manager, I don't think it's worth it to mention ` using the Content API.`